docs/content/en/hugo-pipes/transpilejs.md

```diff
@@ -0,0 +1,69 @@
+---
+title: TranspileJS
```

Comment:
I personally would maybe name it by name similar to postcss like `babel`, because there are also other tools that do similar things. For example `bublè` (https://github.com/bublejs/buble). But babel is the de facto standard so it might also make sense to just go with `TranspileJS` and not worry about alternatives.
Not sure.